packages/cosmic-proto/test/address-hooks.test.js

```diff
@@ -0,0 +1,214 @@
+import rawTest from '@endo/ses-ava/prepare-endo.js';
+
+import bundleSourceAmbient from '@endo/bundle-source';
+import { importBundle } from '@endo/import-bundle';
+
+/**
+ * @type {import('ava').TestFn<{
+ *   addressHooks: import('../src/address-hooks.js');
+ * }>}
+ */
+const test = rawTest;
+
+const makeTestContext = async () => {
```

Comment:
Why all the `importBundle` gymnastics? I suppose it's to test that `address-hooks.js` works in a compartment. If so, a 1-line comment to that effect seems cost-effective.